docs/src/Checkbox.doc.js

```diff
@@ -122,37 +115,35 @@ card(
   "
     name="Example: Labeled stack"
     defaultCode={`
-class CheckboxExample extends React.Component {
-  render() {
-    const CheckboxWithLabel = ({ id, label }) => (
-      <Box alignItems="center" direction="row" display="flex">
-        <Checkbox
-          checked
-          id={id}
-          onChange={() => {}}
-        />
-        <Label htmlFor={id}>
-          <Box paddingX={2}>
-            <Text>{label}</Text>
-          </Box>
-        </Label>
-      </Box>
-    );
-
-    return (
-      <Box display="flex" direction="column" justifyContent="around" marginTop={-1} marginBottom={-1}>
-        <Box paddingY={1}>
-          <CheckboxWithLabel label="Email" id="email" />
-        </Box>
-        <Box paddingY={1}>
-          <CheckboxWithLabel label="Mobile push" id="push" />
-        </Box>
-        <Box paddingY={1}>
-          <CheckboxWithLabel label="Carrier pidgeon" id="pidgeon" />
+function CheckboxExample() {
+  const CheckboxWithLabel = ({ id, label }) => (
```

Comment:
This is a bit odd but `react-live` breaks if we extract `CheckboxWithLabel` out of `CheckboxExample` since it does an automatic `return` on whatever is passed into `defaultCode`